source/src/molecular-unfolding/cdk/stack-main.ts

```diff
@@ -35,41 +40,38 @@ import {
   SolutionStack,
 } from '../../stack';
 
-
-import {
-  BatchEvaluation,
-} from './construct-batch-evaluation';
-
-import {
-  Dashboard,
-} from './construct-dashboard';
-
-import {
-  EventListener,
-} from './construct-listener';
 import {
   Notebook,
 } from './construct-notebook';
-
-import create_custom_resources from './utils/custom-resource';
+import {
+  BatchEvaluationNestStack,
+} from './statck-batch-evaluation';
+import {
+  VisualizationNestStack,
+} from './statck-visualization';
 import {
   AddCfnNag,
-  AddCondition,
-  ChangePolicyName,
 } from './utils/utils';
+
 import setup_vpc_and_sg from './utils/vpc';
 
 export class MainStack extends SolutionStack {
   static SOLUTION_ID = 'SO8027'
   static SOLUTION_VERSION = process.env.SOLUTION_VERSION || 'v1.0.0'
+  static DESCRIPTION = `(${MainStack.SOLUTION_ID}) Quantum Computing Exploration for Drug Discovery on AWS ${MainStack.SOLUTION_VERSION}`;
 
   // constructor
   constructor(scope: Construct, id: string, props: StackProps = {}) {
-    const DESCRIPTION = `(${MainStack.SOLUTION_ID}) Quantum Computing Exploration for Drug Discovery on AWS ${MainStack.SOLUTION_VERSION}`;
+
     super(scope, id, props);
-    this.setDescription(DESCRIPTION);
+    this.setDescription(MainStack.DESCRIPTION);
     const stackName = this.stackName.replace(/[^a-zA-Z0-9_]+/, '').toLocaleLowerCase();
 
+
+    const cfnTemplate = new CfnInclude(this, 'CfnTemplate', {
+      templateFile: path.join(__dirname, 'parameter-group.template'),
```

Comment:
why putting it as a separated cfn file? CDK also supports those L1 interfaces.

Reply:
need to group parameters.

Reply:
refer to this,

https://github.com/aws/aws-cdk/issues/5944#issuecomment-581365345

Reply:
changed to templateOptions.metadata.